### PR TITLE
Adds post on audit communit report

### PR DESCRIPTION
**Pull request by bmbouter**

### CODE REVIEW

**Comment by bmbouter** [comment]
Rendered version is available here:  https://bmbouter.github.io/pulpproject.org/2018/09/17/pulp-community-health-audit/

File: _posts/2018-09-17-pulp-community-health-audit.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+This blog gives a summary, but you can read the full report
+[here](/files/Community_Audit_Report_Pulp_2018.pdf).
+
+
+# Overview
+
+Overall the Pulp community is healthy, but it
+can improve by making most of its resources more clearly discoverable, e.g. quick start guides, etc.
+Here are the recommended actions for the Pulp community to do to improve itself from the audit:
```
Comment (daviddavis) [inline _posts/2018-09-17-pulp-community-health-audit.md]: Here are the recommended actions from the audit for how the Pulp community can improve:

File: _posts/2018-09-17-pulp-community-health-audit.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+towards previous goals.
+
+This blog gives a summary, but you can read the full report
+[here](/files/Community_Audit_Report_Pulp_2018.pdf).
+
+
+# Overview
+
+Overall the Pulp community is healthy, but it
+can improve by making most of its resources more clearly discoverable, e.g. quick start guides, etc.
```
Comment (daviddavis) [inline _posts/2018-09-17-pulp-community-health-audit.md]: making more of its resources

File: _posts/2018-09-17-pulp-community-health-audit.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+Here are the recommended actions for the Pulp community to do to improve itself from the audit:
+
+* Create a quick start documentation guide
+* Create a more visible link to the PUPs
+* Create an online calendar to create a steady-state place for community members to discover events
```
Comment (daviddavis) [inline _posts/2018-09-17-pulp-community-health-audit.md]: s/to create/as/

File: _posts/2018-09-17-pulp-community-health-audit.md [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+* Create a direct link to release notes on the Pulp web site
+* Review PUPs to see if a Tech Advisory Board or other adjustments need to be made to governance
```
Comment (daviddavis) [inline _posts/2018-09-17-pulp-community-health-audit.md]: /to governance/to the governance/

File: _posts/2018-09-17-pulp-community-health-audit.md [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+  model
+* Explore if Pulp would benefit from a part-time or full-time outreach and marketing person
+
+
+# Infrastructure and Process
+
+Recommendations to Pulp come in two areas here: Resource Discoverability and Process Improvements.
+
+Pulp needs to focus on its website and docs some to ensure users can find what they need quickly.
```
Comment (daviddavis) [inline _posts/2018-09-17-pulp-community-health-audit.md]: s/some//

File: _posts/2018-09-17-pulp-community-health-audit.md [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+For example, this comes through more quickstart guides, and fewer clicks required to get users to
```
Comment (daviddavis) [inline _posts/2018-09-17-pulp-community-health-audit.md]: Some suggested improvements include more quickstart guides and fewer page clicks for users to get to key pages such as the installation page.

File: _posts/2018-09-17-pulp-community-health-audit.md [hunk 7]
```diff
@@ -0,0 +1,83 @@
...
+# Activity
+
+There isn't data available on the commit history or contributor activity or engagement in the
+report. In terms of community events, Pulp has no public calendar showing all of the events a user
+or contributor could get involved with.
+
+
+# Documentation
+
+Pulp needs to become easier ot install, and as it does its documentation will get shorter. Pulp also
```
Comment (daviddavis) [inline _posts/2018-09-17-pulp-community-health-audit.md]: s/ot/to/

File: _posts/2018-09-17-pulp-community-health-audit.md [hunk 8]
```diff
@@ -0,0 +1,83 @@
...
+needs better organized documentation, along with quick-start assets. In some cases the documentation
+is available, but not discoverable, e.g. the release notes were hard to find.
```
Comment (daviddavis) [inline _posts/2018-09-17-pulp-community-health-audit.md]: s/were/are/

File: _posts/2018-09-17-pulp-community-health-audit.md (hunk 1 above)
Comment (bmbouter) [reply _posts/2018-09-17-pulp-community-health-audit.md]: ty fixed in next push.

File: _posts/2018-09-17-pulp-community-health-audit.md (hunk 2 above)
Comment (bmbouter) [reply _posts/2018-09-17-pulp-community-health-audit.md]: ty fixed in next push.

File: _posts/2018-09-17-pulp-community-health-audit.md [hunk 9]
```diff
@@ -0,0 +1,83 @@
...
+
+# Code Quality
+
+Pulp will re-assess its code-quality metrics with Pulp3 which has significantly different code line
+count for its feature set relative to Pulp2.
+
+
+# Outreach
+
+Outreach such as its blog, social media, and conference activity is one area Pulp can do much more.
```
Comment (daviddavis) [inline _posts/2018-09-17-pulp-community-health-audit.md]: s/its blog/blog/

File: _posts/2018-09-17-pulp-community-health-audit.md (hunk 4 above)
Comment (bmbouter) [reply _posts/2018-09-17-pulp-community-health-audit.md]: ty fixed in next push.

File: _posts/2018-09-17-pulp-community-health-audit.md (hunk 5 above)
Comment (bmbouter) [reply _posts/2018-09-17-pulp-community-health-audit.md]: ty

File: _posts/2018-09-17-pulp-community-health-audit.md [hunk 10]
```diff
@@ -0,0 +1,83 @@
...
+For the blog, Pulp needs to have more, and more regular, activity on its blog to increase its
```
Comment (daviddavis) [inline _posts/2018-09-17-pulp-community-health-audit.md]: For the blog, Pulp needs to have more activity on a more consistent basis to increase its engagement with users.

File: _posts/2018-09-17-pulp-community-health-audit.md [hunk 11]
```diff
@@ -0,0 +1,83 @@
...
+engagement with its users. Similarly for its social media, we want to have more engagement there.
```
Comment (daviddavis) [inline _posts/2018-09-17-pulp-community-health-audit.md]: for social media

File: _posts/2018-09-17-pulp-community-health-audit.md (hunk 6 above)
Comment (bmbouter) [reply _posts/2018-09-17-pulp-community-health-audit.md]: ty

File: _posts/2018-09-17-pulp-community-health-audit.md [hunk 12]
```diff
@@ -0,0 +1,83 @@
...
+Additionally Pulp needs to increase its visibility at user-facing conferences.
+
+
+# License
+
+Pulp maybe should consider a Contributor License Agreement, but the last time open source lawyers
+advised us not have a CLA was ok.
```
Comment (daviddavis) [inline _posts/2018-09-17-pulp-community-health-audit.md]: s/have/having/

File: _posts/2018-09-17-pulp-community-health-audit.md (hunk 3 above)
Comment (bmbouter) [reply _posts/2018-09-17-pulp-community-health-audit.md]: :palm_tree: :+1:

**Review by daviddavis — APPROVED** [review]
Left some suggestions. Feel free to use/discard whichever ones make sense.

File: _posts/2018-09-17-pulp-community-health-audit.md (hunk 7 above)
Comment (bmbouter) [reply _posts/2018-09-17-pulp-community-health-audit.md]: ty, fixed in next push.

File: _posts/2018-09-17-pulp-community-health-audit.md (hunk 8 above)
Comment (bmbouter) [reply _posts/2018-09-17-pulp-community-health-audit.md]: fixed in next push

File: _posts/2018-09-17-pulp-community-health-audit.md (hunk 9 above)
Comment (bmbouter) [reply _posts/2018-09-17-pulp-community-health-audit.md]: fixed

File: _posts/2018-09-17-pulp-community-health-audit.md (hunk 10 above)
Comment (bmbouter) [reply _posts/2018-09-17-pulp-community-health-audit.md]: yup

File: _posts/2018-09-17-pulp-community-health-audit.md (hunk 11 above)
Comment (bmbouter) [reply _posts/2018-09-17-pulp-community-health-audit.md]: fixed in next push

File: _posts/2018-09-17-pulp-community-health-audit.md (hunk 12 above)
Comment (bmbouter) [reply _posts/2018-09-17-pulp-community-health-audit.md]: yup, fixed in next push

**Comment by bmbouter** [comment]
Thanks @daviddavis I made all the edits! I'm going to merge this now. I really appreciate it.

**Pull request merged**